### ISSUE TITLE
[Feature]: Pronouns in profile

### ISSUE THREAD

**Issue by Mikaela** [simplex-chat/simplex-chat#2553]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### App version

5.1.2

### Feature

I would like to have the ability to set pronouns in a field separate from the name and display name. Somehow like GitHub does it, that the information is readily available by hovering over my profile or behind touch in the app (I am not sure if that differs on whether viewing the profile as someone else).

I am not sure whether it should be per-group or global option, which would probably be fine with there being support for multiple profiles already.

Another part I am unsure of is incognito mode, I can understand desiring an option to specify pronouns that also stuck in incognito mode with the cost of one point of anonymity, but I also really like the idea of incognito mode generating pronouns, possibly even neopronouns, for me.

- - - - -

_This request is inspired by another communication platform where pronoun support isn't moving forwards and where having pronouns in display name is frequently infrequent point of conflict and where recently abusive users decided that everyone having pronouns in their name must belong to a gender minority inspires me to visualise incognito mode with everyone having random pronouns._

**Comment by amosstevens** [comment]
in my point of view that seems a too specific issue with a specific case, I think it would be good to have a generalization of that

like, adding the ability to include a profile description or tags, visible to other users

unless it would compromise the anonymity or security of the app

**Comment by kimg45** [comment]
Yeah you can already just put your pronouns in your name if you want. No need for a dedicated thing for it. Or have some kind of profile description that you can put any text in.

**Comment by kevATin** [comment]
I don't think there need to be dedicated fields for preferred pronouns.
But I've now opened a suggestion for Profile Descriptions #5079 where users could easily place such information. And the description would be much more versatile in use, so each user can decide for themselves what they want to put there.

**Comment by EchedelleLR** [comment]
That would hide visibility to it, as the idea of having a specific field is that.

The name is also a no-go for obvious reasons.
